sklearn/lda.py

```diff
@@ -391,6 +397,12 @@ def fit(self, X, y, store_covariance=False, tol=1.0e-4):
 
         y : array, shape (n_samples,)
             Target values.
+
+        store_covariance : bool, optional
```

Comment:
The reason they got picked up is because the default value was not set to ``None`` when they were deprecated, which was an oversight. If you like you can do that (and if they are not ``None`` set them to the correct value).